non-oss/pharos_pro/addons/kontena-lens/addon.rb

```diff
@@ -70,10 +70,11 @@
     protocol = tls_enabled? ? 'https' : 'http'
     message = "Kontena Lens is configured to respond at: " + "#{protocol}://#{host}".cyan
     message << "\nStarting up Kontena Lens the first time might take couple of minutes, until that you'll see 503 with the address given above."
+    kubernetes_api_endpoint = cluster_config.api&.endpoint || "https://#{master_host_ip}:6443"
```

Comment:
Currently `api.endpoint` does not contain protocol or port.

Reply:
can we assume that protocol is `https`?

Reply:
We have to make same assumptions as with `master_host_ip`.

Reply:
but port is standard `443`?

Reply:
> but port is standard 443?

No, as I said we need to make same assumptions .. that includes the port.